apps/graphql/src/apps/location/types/outputs/Location.js

```diff
@@ -32,6 +32,9 @@ export default new GraphQLObjectType({
     coordinates: {
       type: GraphQLCoordinate,
     },
+    type: {
+      type: GraphQLString,
```

Comment:
Maybe it should be an enum?

Reply:
I was thinking about that, but I can't find any documentation about all possible values from TequilaAPI. But I asked for that so we will see.